components/navbar/navbar-left.vue

```diff
@@ -1,33 +1,44 @@
 <template>
   <nav class="flex flex-col mt-16">
-    <nuxt-link class="mb-2 p-1 font-semibold" to="/">
-      <span class="pb-1">Home</span>
-    </nuxt-link>
-    <nuxt-link class="mb-2 p-1 font-semibold" to="/#">
-      <span class="pb-1">Sponsoring us</span>
-    </nuxt-link>
-    <nuxt-link class="mb-2 p-1 font-semibold" to="/#">
-      <span class="pb-1">Gallery</span>
+    <nuxt-link
+      v-for="n in getNav"
+      :to="localePath(n.real_path)"
+      class="mb-2 p-1 font-semibold"
+      exact
+    >
+      <span class="pb-1">{{ n.name }}</span>
     </nuxt-link>
     <div class="mt-16 p-1">
       <div class="mb-2 font-semibold">Lang</div>
       <div>
-        <button class="mr-2 px-1 pt-2 border-2 border-gray-500">
-          <img src="~/assets/flag-us.png" alt="" />
-          US
-        </button>
-        <button class="mr-2 px-1 pt-2 border-2 border-gray-500">
-          <img src="~/assets/flag-fr.png" alt="" />
-          FR
+        <button
+          v-for="locale in availableLocales"
+          :key="locale"
+          @click="$i18n.setLocale(locale)"
+          :class="
+            $i18n.locale === locale ? 'border-green-500' : 'border-gray-500'
+          "
+          class="mr-2 px-1 pt-2 border-2 "
+        >
+          <img :src="require(`~/assets/${locale}.png`)" alt="" />
+          {{ locale }}
         </button>
       </div>
     </div>
   </nav>
 </template>
 
 <script>
+import { mapGetters } from 'vuex'
+
 export default {
-  name: 'NavBarLeft'
+  name: 'NavBarLeft',
+  computed: {
+    ...mapGetters(['getNav']),
+    availableLocales() {
+      return this.$i18n.locales.filter((i) => i.code !== this.$i18n.locale)
```

Comment:
NIT but I prefer be obvious with `i` like
``` javascript
return this.$i18n.locales.filter((something) => something.code !== this.$i18n.locale)
```
But maybe it's just me 😸

Reply:
i agree